### ISSUE TITLE
AWS test failing with GRUB2_ROOT_FS_UUID error

### ISSUE THREAD

**Issue by major** [osbuild/osbuild-composer#558]
The AWS image tests in #552 are failing with this error:

```
                	            	Diff:
                	            	--- Expected
                	            	+++ Actual
                	            	@@ -1,4 +1,3 @@
                	            	 (map[string]interface {}) (len=16) {
                	            	- (string) (len=16) "boot-environment": (map[string]interface {}) (len=2) {
                	            	-  (string) (len=18) "GRUB2_ROOT_FS_UUID": (string) (len=36) "76a22bf4-f153-4541-b6c7-0332c0dfaeac",
                	            	+ (string) (len=16) "boot-environment": (map[string]interface {}) (len=1) {
                	            	   (string) (len=10) "kernelopts": (string) (len=149) "root=UUID=76a22bf4-f153-4541-b6c7-0332c0dfaeac ro no_timer_check console=ttyS0,115200n8 console=tty1 biosdevname=0 net.ifnames=0 console=ttyS0,115200"
                	Test:       	TestImages//usr/share/tests/osbuild-composer/cases/fedora_31-x86_64-ami-boot.json/image_info
        --- PASS: TestImages//usr/share/tests/osbuild-composer/cases/fedora_31-x86_64-ami-boot.json/boot (584.89s)
```

Jenkins job: https://jenkins.osbuild.org/job/osbuild-composer-fedora31-aws/22/

**Comment by teg** [comment]
This is likely a fallout of https://github.com/osbuild/osbuild/pull/335/, though I think the solution should be to update the test.

cc @gicmo

**Comment by gicmo** [comment]
Yeah, mentioned that on the chat yesterday already. Was removed on purpose. Test should be adapted.

**Comment by ondrejbudai** [comment]
Fixed by #573
